### DISCUSSION TITLE
.net 6 7and vs2022 support

### DISCUSSION THREAD

**Discussion by juepiezhongren** [cefsharp/CefSharp#4536] [General]
is this available now?

**Comment by amaitland** [comment]
In future please use [Discussions](https://github.com/cefsharp/CefSharp/discussions) to ask support related questions.

See for basic instructions https://github.com/cefsharp/CefSharp/wiki/Quick-Start-For-MS-.Net-5.0-or-greater

There's a working example provided https://github.com/cefsharp/CefSharp.MinimalExample#cefsharpminimalexample
